README.md

```diff
@@ -35,5 +35,24 @@ for dir in examples/*; do
   echo "$dir/coq-${dir#examples/}.opam"
 done
 ```
+
+You can generate the standard files from the templates,
+provided you already have written `meta.yml`, like this:
+```shell
+git clone git@github.com:coq-community/templates.git
+cd <your_coq_project>
+ln -sf ../templates/{,.}*.mustache .
+cat << . > ./generate.sh
```

Comment:
Hi @amiloradovsky ! This looks great. Is there any reason not to factor this part as a standalone script and add it as a file to the repo?

Reply:
OK, done.